jetty-http-spi/src/main/java/org/eclipse/jetty/http/spi/JettyHttpExchangeDelegate.java

```diff
@@ -209,8 +209,15 @@ public void setAttribute(String name, Object value)
     @Override
     public void setStreams(InputStream i, OutputStream o)
     {
-        _is = i;
-        _os = o;
+        assert _is != null;
+        if (i != null)
+        {
+            _is = i;
+        }
+        if (o != null)
+        {
+            _os = o;
+        }
```

Comment:
Drop the braces in case of one-liners.

```suggestion
        if (i != null)
            _is = i;
        if (o != null)
            _os = o;
```